includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -132,15 +132,23 @@ public function stripe_check_environment() {
 			if ( empty( $show_3ds_notice ) && $three_d_secure ) {
 				$url = 'https://stripe.com/docs/payments/3d-secure#three-ds-radar';
 
-				/* translators: 1) A URL that explains Stripe Radar. */
-				$message = __( 'WooCommerce Stripe - We see that you had the "Require 3D secure when applicable" setting turned on. This setting is not available here anymore, because it is now replaced by Stripe Radar. You can learn more about it <a href="%s" target="_blank">here</a>.', 'woocommerce-gateway-stripe' );
-
-				$this->add_admin_notice( '3ds', 'notice notice-warning', sprintf( $message, $url ), true );
+				$message = sprintf(
+				/* translators: 1) HTML anchor open tag 2) HTML anchor closing tag */
+					__( 'WooCommerce Stripe - We see that you had the "Require 3D secure when applicable" setting turned on. This setting is not available here anymore, because it is now replaced by Stripe Radar. You can learn more about it %1$shere%2$s ', 'woocommerce-gateway-stripe' ),
+					'<a href="' . $url . '" target="_blank">',
+					'</a>'
+				);
+
+				$this->add_admin_notice( '3ds', 'notice notice-warning', $message, true );
 			}
 
 			if ( empty( $show_style_notice ) ) {
-				/* translators: 1) int version 2) int version */
-				$message = __( 'WooCommerce Stripe - We recently made changes to Stripe that may impact the appearance of your checkout. If your checkout has changed unexpectedly, please follow these <a href="https://woocommerce.com/document/stripe/#new-checkout-experience" target="_blank">instructions</a> to fix.', 'woocommerce-gateway-stripe' );
+				$message = sprintf(
+				/* translators: 1) HTML anchor open tag 2) HTML anchor closing tag */
+					esc_html__( 'WooCommerce Stripe - We recently made changes to Stripe that may impact the appearance of your checkout. If your checkout has changed unexpectedly, please follow these %1$sinstructions%2$s to fix.', 'woocommerce-gateway-stripe' ),
```

Comment:
Looks like we don't need to `esc_html()` for messages that are added via `$this->add_admin_notice` because of what we chatted about [here](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210#discussion_r761409998).

Other instances are:
[1](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR195)
[2](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR211)
[3](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR226)
[4](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR242)
[5](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR256)
[6](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR268)
[7](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2210/files#diff-157591db949a6ea264b2394080349c5374732bc491c5e5fbf7637351ab556a3fR279)

Reply:
Fixed by f768b8f7e997501b71cc071e4f12135d7ba0b4ef